d3a_interface/sim_results/aggregate_results.py

```diff
@@ -66,29 +66,30 @@ def _merge_target_area_unmatched_loads(cls, accumulated_results: Dict,
         :param target: target area of the accumulated unmatched loads
         :return: None
         """
-        target_ul = accumulated_results[area][target]['unmatched_loads']
-        current_ul = current_results[area][target]['unmatched_loads']
-        for timestamp, ts_value in current_ul.items():
-            if timestamp not in target_ul:
-                target_ul[timestamp] = deepcopy(ts_value)
-            else:
-                if 'unmatched_times' not in current_ul[timestamp]:
-                    continue
-                if 'unmatched_times' not in target_ul[timestamp]:
-                    target_ul[timestamp]['unmatched_times'] = {}
-                for device, time_list in current_ul[timestamp]['unmatched_times'].items():
-                    if device not in target_ul[timestamp]['unmatched_times']:
-                        target_ul[timestamp]['unmatched_times'][device] = deepcopy(time_list)
-                    else:
-                        for ts in time_list:
-                            if ts not in target_ul[timestamp]['unmatched_times'][device]:
-                                target_ul[timestamp]['unmatched_times'][device].append(ts)
-
-                unm_count = 0
-                for _, hours in target_ul[timestamp]['unmatched_times'].items():
-                    unm_count += len(hours)
-
-                    target_ul[timestamp]['unmatched_count'] = unm_count
+        target_ul = accumulated_results[area][target].get('unmatched_loads', "")
+        current_ul = current_results[area][target].get('unmatched_loads', "")
+        if target_ul != "" and current_ul != "":
```

Comment:
It is probably minor, but how about using `None` for the default value and checking for `None` in the if clause?